packages/core/mesh/network-manager/src/testing/test-builder.ts

```diff
@@ -77,48 +71,24 @@ export class TestPeer {
   private _proxy?: ProtoRpcPeer<any>;
   private _service?: ProtoRpcPeer<any>;
 
-  constructor(private readonly testBuilder: TestBuilder, public readonly peerId: PublicKey) {
+  constructor(
+    private readonly testBuilder: TestBuilder,
+    public readonly peerId: PublicKey,
+    public readonly webrtcLibrary?: string,
+  ) {
     this._signalManager = this.testBuilder.createSignalManager();
-    this._networkManager = this.createNetworkManager();
+    this._networkManager = this.createNetworkManager(webrtcLibrary);
   }
 
   // TODO(burdon): Move to TestBuilder.
-  createNetworkManager() {
+  createNetworkManager(webrtcLibrary?: string) {
     let transportFactory: TransportFactory = MemoryTransportFactory;
 
     if (this.testBuilder.options.signalHosts) {
       if (this.testBuilder.options.bridge) {
-        // Simulates bridge to shared worker.
-        const [proxyPort, servicePort] = createLinkedPorts();
-
-        this._proxy = createProtoRpcPeer({
-          port: proxyPort,
-          requested: {
-            BridgeService: schema.getService('dxos.mesh.bridge.BridgeService'),
-          },
-          noHandshake: true,
-          encodingOptions: {
-            preserveAny: true,
-          },
-        });
-
-        this._service = createProtoRpcPeer({
-          port: servicePort,
-          exposed: {
-            BridgeService: schema.getService('dxos.mesh.bridge.BridgeService'),
-          },
-          handlers: { BridgeService: new WebRTCTransportService() },
-          noHandshake: true,
-          encodingOptions: {
-            preserveAny: true,
-          },
-        });
-
-        transportFactory = new WebRTCTransportProxyFactory().setBridgeService(this._proxy.rpc.BridgeService);
+        throw new Error('no longer implemented.');
```

Comment:
what's the reason for ripping those out? they are still useful

Reply:
due to not understanding the purpose, will restore and fix once separating the factory methods